src/core/server/ui_settings/integration_tests/lib/servers.ts

```diff
@@ -6,13 +6,16 @@
  * Side Public License, v 1.
  */
 
+import supertest from 'supertest';
```

Comment:
```suggestion
import type supertest from 'supertest';
```

Reply:
otherwise, it's easy to start using it instead of the pre-configured instance provided by `getSupertest`